test/utilities.jl

```diff
@@ -35,6 +35,17 @@ export OuterModule
 end
 end
 
+module ModuleWithAlias
+using LinearAlgebra
```

Comment:
Create another test module instead? I believe this will pull in `LinearAlgebra` and `Random` as dependencies, which is not optimal.

Reply:
Done. I do think there should be a module that's its own parent as a test case, so I'm using `Main` for that now.